src/test/java/teammates/test/cases/action/FeedbackParticipantFeedbackResponseCommentDeleteActionTest.java

```diff
@@ -0,0 +1,386 @@
+package teammates.test.cases.action;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.FeedbackQuestionAttributes;
+import teammates.common.datatransfer.attributes.FeedbackResponseAttributes;
+import teammates.common.datatransfer.attributes.FeedbackResponseCommentAttributes;
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.util.Const;
+import teammates.storage.api.FeedbackQuestionsDb;
+import teammates.storage.api.FeedbackResponseCommentsDb;
+import teammates.storage.api.FeedbackResponsesDb;
+import teammates.ui.controller.AjaxResult;
+import teammates.ui.controller.FeedbackParticipantFeedbackResponseCommentDeleteAction;
+import teammates.ui.pagedata.FeedbackResponseCommentAjaxPageData;
+
+/**
+ * SUT: {@link FeedbackParticipantFeedbackResponseCommentDeleteAction}.
+ */
+public class FeedbackParticipantFeedbackResponseCommentDeleteActionTest extends BaseActionTest {
+
+    @Override
+    protected String getActionUri() {
+        return Const.ActionURIs.FEEDBACK_PARTICIPANT_FEEDBACK_RESPONSE_COMMENT_DELETE;
+    }
+
+    @Override
+    protected void prepareTestData() {
+        super.prepareTestData();
+        dataBundle = loadDataBundle("/FeedbackParticipantFeedbackResponseCommentDeleteTest.json");
+        removeAndRestoreDataBundle(dataBundle);
+    }
+
+    @Override
+    @Test
+    public void testExecuteAndPostProcess() {
+        testFailureCases();
+        testDeleteActionForInstructorAsFeedbackParticipant();
+        testDeleteActionForStudentAsFeedbackParticipant();
+        testDeleteActionForTeamAsFeedbackParticipant();
+    }
+
+    private void testFailureCases() {
+        FeedbackResponseCommentAttributes feedbackResponseComment = getCommentFromInstructor1();
```

Comment:
All of the tests in this file are independent of each other and can be passed by adding `@Test` on top of function. I was wondering if it is okay to leave `testExecuteAndPostProcess()` and `testAccessControl() ` empty and make these as unit tests.